blockly-core/build-output/blockly_uncompressed.js

```diff
@@ -23437,7 +23437,7 @@ Blockly.BlockSpaceEditor.prototype.createDom_ = function(container) {
   svg.appendChild(this.blockSpace.createDom());
```

Comment:
Okay.  I didn't know our build process took care of this.  I can back these out then.